### PR TITLE
Fix CLI start command to allow passing variadic arguments in a civilized way

### PR DESCRIPTION
**Pull request by pietrzakacper**
Before we had to pass arguments to the start command like this:
```
si seq start 27cb6496-5895-40a9-9087-7a3b4c26253a '["BTC, "ETH"]'
```

now it's possible to do it the normal way:

```
si seq start 27cb6496-5895-40a9-9087-7a3b4c26253a BTC ETH
```

and the bonus is, we support passing no arguments ;)

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "si": "./src/bin/index.ts"
   },
   "scripts": {
-    "start": "ts-node ./src/index",
+    "start": "ts-node ./src/bin/index",
```
Comment (jan-warchol) [inline packages/cli/package.json:10]: why this change? it seems unrelated fo arg parsing.

**Review by jan-warchol — COMMENTED** [review]
And that was it? Nice!

File: packages/cli/package.json (hunk 1 above)
Comment (pietrzakacper) [reply packages/cli/package.json:10]: it's needed to run and test this package locally

Comment (pietrzakacper) [reply packages/cli/package.json:10]: ./src/index file does not exists ;)

**Review by MichalCz — APPROVED** [review]

File: packages/cli/package.json (hunk 1 above)
Comment (jan-warchol) [reply packages/cli/package.json:10]: ah, so it was broken from the start? pity.
